src/components/Checkbox/index.css

```diff
@@ -0,0 +1,8 @@
+.checkbox {
```

Comment:
```suggestion
input[type=checkbox] {
```
instead of styling the check boxes using `classNames` you can do this